### PR TITLE
Add HSS1394Config.cmake.in file

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -64,18 +67,45 @@ endif()
 # Installation
 #
 
-include(GNUInstallDirs)
```
Comment (Be-ing) [inline CMakeLists.txt:67]: Why did you remove this?

Comment (Holzhaus) [reply CMakeLists.txt:67]: Moved to line 9 because it's already needed in line 22 :)
